indexer/feature_altitude.cpp

```diff
@@ -0,0 +1,58 @@
+#include "indexer/feature_altitude.hpp"
+
+#include "base/bits.hpp"
+
+namespace feature
+{
+void Altitudes::PrepareSerializationData(TAltitude minAltitude, vector<uint32_t> & deviations) const
+{
+  CHECK(!m_altitudes.empty(), ());
+
+  deviations.clear();
+  uint32_t const firstPntDeviation =
+      bits::ZigZagEncode(static_cast<int32_t>(m_altitudes[0]) -
+      static_cast<int32_t>(minAltitude)) + 1 /* making it more than zero */;
+  CHECK_LESS(0, firstPntDeviation, ());
+  deviations.push_back(firstPntDeviation);
+  for (size_t i = 1; i < m_altitudes.size(); ++i)
+  {
+    uint32_t const nextPntDeviation = bits::ZigZagEncode(static_cast<int32_t>(m_altitudes[i]) -
+        static_cast<int32_t>(m_altitudes[i - 1])) + 1 /* making it more than zero */;
```

Comment:
Please, remove these +1 from the code. This is the problem of delta coder that it can't handle zero. All these +1/-1 should be moved to the place where delta coder is used.